content/projects/kinfraglib_pocket_enum.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Novel Kinase Ligand Generation using Subpocket-Based Docking"
+date: 2023-05-05
+nav: false
+weight: 10
+# Uncomment sections below to enable them in your project
+#external_resources:
+#- name: KinFragLib_PocketEnum Repository
+#  link: https://github.com/volkamerlab/KinFragLib_PocketEnum
+#  icon: github
+people:
+- key: katharina.buchthal
+- key: paula.kramer
+- key: andrea.volkamer
+---
+
+The majority of signal transduction in eukaryotic cells is mediated by protein kinases, whereas dysregulation is often associated with cancer. Fragment-based drug design strategies have already shown promising results for developing novel kinase inhibitors. Typically, these approaches consider the structural information about the target but do not take the kinase-specific prior knowledge into account, although the kinase binding pocket is highly conserved and well-studied.
```

Comment:
> Typically, these approaches consider the structural information about the target but do not take the kinase-specific prior knowledge into account, although the kinase binding pocket is highly conserved and well-studied.

Try to rephrase the sentence to ensure that it is clear that we also use only information from ligand site, i.e. KinFragLib fragments